plugins/catalog-react/src/components/EntityDisplayName/EntityDisplayName.tsx

```diff
@@ -77,7 +86,9 @@ export const EntityDisplayName = (
   );
 
   // The innermost "body" content
-  let content = <>{primaryTitle}</>;
+  let content = (
+    <Typography className={classes.truncate}>{primaryTitle}</Typography>
```

Comment:
There is a `noWrap` property that could be used this purpose, If that works, we could skip the CSS changes 🙂 
https://v4.mui.com/api/typography/#typography-api